source/partials/next_release/_release.md.erb

```diff
@@ -1,87 +1,88 @@
-#### API Enhancements
+<h4>Authorization Plugin</h4>
 
-* <%= link_to_commit 'f47d054171b07e2ce9c2efa9e02dde6979fc9c8f', "#{link_to_api 'encryption'} to encrypt secrets.", 1829 %>
+There has been more work done around fine tuning the Authorization Plugin endpoints and as part of this some of the plugin API calls have changed.
 
-#### Other Improvements
+Action required: Early adopters of this extension will have to update their plugins to handle the updated API calls before upgrading Go. Refer to the Authorization Plugin Endpoint [documentation](https://plugin-api.gocd.io/current/authorization) for the latest changes.
 
-* <%= link_to_commit 'xxx', 'YYY.', 123 %>
+The built-in LDAP and password file based authentication mechanisms have been deprecated in favor of separate plugins that provide more functionality, with the benefit of allowing users to customize the behavior of these plugins. These plugins are bundled with the GoCD server, so no action will be required from users.
 
-#### Bug Fixes
+* LDAP authentication plugin
 
-* <%= link_to_commit 'e9611f83798bd5eddd62326f0827e5055519bd4a', 'OS Detection failing for elastic agents.', 3060 %>
-* Multiple UI fixes and tweaks on the new theme that was introduced in 16.12.
-* <%= link_to_commit 'ad1a1188d15839ac4a41c91c84c631612e49bf60', 'Sort resources and environments alphabetically on agents page selector dropdown.', 3134 %>
+  The bundled [LDAP authentication plugin](https://github.com/gocd/gocd-ldap-authentication-plugin) will now allow users to connect to and authenticate with multiple LDAP servers in their organizations.
 
-#### Breaking changes
+* Password File Authentication Plugin
 
-* <%= link_to_commit 'xxx', 'YYY', 456 %>
+  The bundled [Password File Authentication Plugin](https://github.com/gocd/filebased-authentication-plugin) currently supports only SHA1 hashed passwords, but support for other hashing functions is [on its way](https://github.com/gocd/filebased-authentication-plugin/issues/13).
```

Comment:
I ask this because I dont see anyone really working on it at this point. And when we do, we do have to worry about password files stored in old formats too, so its not just a simple change in algorithm. Sorry, if I sound demotivating.